ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx

```diff
@@ -51,7 +51,7 @@ const getPageData = async ({
 
   const env = getEnvironment(pathname);
   const optHeaders = { 'ctx-service-env': env };
-  const isLocal = !env || env === 'local';
+  const isLocal = !env || env === 'local' || !!process.env.INTEGRATION;
```

Comment:
I am finding this one difficult to read - is there any way we can simplify this? Or, can we just ensure that the env is set to local in these integration tests, rather than having an environment variable where `INTEGRATION=true`

Reply:
I think `env === 'local' || process.env.INTEGRATION;` might do the trick.
env==='local' embeds !env because then env is null or undefined then env==='local' is also false

I am not sure what the !! is used for, would this just be itself?